assets/dev/js/editor/views/add-section/base.js

```diff
@@ -30,6 +30,7 @@ class AddSectionBase extends Marionette.ItemView {
 
 	attributes() {
 		return {
+			'aria-label': __( 'Add new layout element', 'elementor' ),
```

Comment:
Change to "Choose layout"

Reply:
@arielk this refers to the wrapping element. This is the text in the accessibility tree.

Inside this wrapper, there are several "screens", not all of them refer to choosing layout.

If you don't mind, I prefer to leave the text as it is.